### PR TITLE
raftstore-v2: supplement read track metrics

### PR DESCRIPTION
**Pull request by SpadeA-Tang**
### What is changed and how it works?

Issue Number: Ref #15409

What's Changed:

```commit-message
supplement read track metrics
```

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test
- Integration test

```release-note
None
```

### CODE REVIEW

**Review by Connor1996 — APPROVED** [review]
LGTM

File: components/raftstore-v2/src/operation/query/local.rs [hunk 1]
```diff
@@ -580,6 +580,10 @@ impl<'r> SnapRequestInspector<'r> {
             ));
         }
 
+        fail::fail_point!("perform_read_index", |_| Ok(ReadRequestPolicy::ReadIndex));
+
+        fail::fail_point!("perform_read_local", |_| Ok(ReadRequestPolicy::ReadLocal));
```
Comment (bufferflies) [inline components/raftstore-v2/src/operation/query/local.rs:585]: where is used

Comment (SpadeA-Tang) [reply components/raftstore-v2/src/operation/query/local.rs:585]: in `test_read_execution_tracking`

**Review by bufferflies — APPROVED** [review]

**Pull request merged**